File: Deberes/01-deber-UniversoAlien/src/Universo.kt
class Universo(
        var nombreUniverso: String,
        var antiguedadUniverso: Int,
        var tamanioUniverso: Float,
        var minTemperatura: Double,
        var universoPrimario:Boolean
){
    //funciones de la clase aqui
    override fun toString ():String{
        return "${nombreUniverso},${antiguedadUniverso},${tamanioUniverso},${minTemperatura},${universoPrimario}\n"
    }

}

File: Deberes/06-CRUD -Con-Salis/Backend-Alien-Universo/universoAlien/api/models/Universo.js
/**
 * Universo.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

   nombreUniverso:{ //nombre del atributo
      type:'String',
      required:true,
      unique:true,
      minLength:10,
      maxLength:25
    },
    antiguedadUniverso:{
      type:'number',
      required:true
    },
    tamanioUniverso:{
      type:'Float',
      required:true 
    },
    minTemperatura:{
      type:'Float',      
      required:true 
    },
    universoPrimario:{
      type:'Boolean',
      required:true
    },

    /*
    Aliens:{//one to many (plural)
      collection: "alien", //referenci al modelo
      via:"universo" //Nombre fk en alien
    }
    */

  },

};

